views/styles.css

```diff
@@ -244,8 +249,18 @@ input[type="text"] {
     }
 }
 
+@media screen and (min-width: 600px) and (max-width: 768px) {
+    .container .navbar {
+        max-width: 600px;
+    }
+}
+
 /*Special sizing for container*/
-@media only screen and (min-width: 600px) and (max-width: 1080px) {
+@media only screen and (min-width: 768px) and (max-width: 1080px) {
```

Comment:
As a suggestion, something we can do about the breakpoints is to stick to a standard. For example bootstrap has fixed breakpoints and when you are designing something, you work around the breakpoints for the most part. 

This maybe something we can use as a guildline for our css - 
- https://getbootstrap.com/docs/5.0/layout/breakpoints/
- https://material-ui.com/customization/breakpoints/